tuxemon/core/components/formula.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
+#
+# Tuxemon
+# Copyright (C) 2014, William Edwards <shadowapex@gmail.com>,
+#                     Benjamin Bean <superman2k5@gmail.com>
+#
+# This file is part of Tuxemon.
+#
+# Tuxemon is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Tuxemon is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Tuxemon.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Contributor(s):
+#
+# Yisroel Newmark <ymnewmark@gmail.com>
+#
+#
+#
+#
+from __future__ import unicode_literals
+import logging
+
+def simple_damage_multiplier(attack_types, target_types):
+    """ Calculates damage multiplier based on strengths and weaknesses
+
+    :param attack_types: The names of the types of the technique.
+    :param target_types: The names of the types of the target.
+
+    :type attack_types: list
+    :type attack_types: list
+
+    :rtype: number
+    :returns: the attack multiplier
+    """
+    m = 1
+    for attack_type in attack_types:
+        if attack_type in None:
```

Comment:
Should be "is None"

This will crash as "None" isn't iterable.

Please test before resubmitting.